part-one/Optimism.md

```diff
@@ -41,26 +39,26 @@ It is our responsibility, both individually and collectively, to make choices ab
+Not all regulation will be good regulation. In fact, the earliest regulation of automotive vehicles was aimed at delaying their adoption by limiting their speed to that of a horse drawn carriage and in some cases even requiring them to be preceded by someone carrying a flag {{"WIKIRF" | cite}}.
 
-Similarly, not all regulation of digital technology will be good regulation. Much of it will initially aim to protect the status quo and help incumbent enterprises. But that is no reason to call for an absence of regulation. It should be seen, instead, as a challenge to come up with the right regulation as we did eventually in the case of cars.
+Similarly, not all regulation of digital technology will be good regulation. Much of it will initially aim to protect the status quo and help incumbent enterprises, such as the recently proposed changes to net neutrality rules {{"NYT3" | cite}}. But that is no reason to call for an absence of regulation. It should be seen, instead, as a challenge to come up with the right regulation as we did eventually in the case of cars.
 
 My proposals for regulation later in the book are aimed at being pro-innovation by giving more economic freedom to individuals and by giving them better access to information (informational freedom). These regulations are choices we need to make collectively. They represent a big departure from the past aimed at letting us explore the space of the possible opened up by digital technologies in a way that can get us past the industrial society to a knowledge society. 
 
 
-## Self Regulation
+## Self-Regulation
 
 There is another set of choices we need to make individually. These have to do with how we react to the massive acceleration of information dissemination and knowledge creation made possible by digital technology. These are not rules society can or should impose because they relate to our inner mental states.
 
-For instance, there are a lot of people at the moment who feel offended by content that is available on the internet. Many comment threads and forums have people yelling at each other. Others spend their time trapped confirming only their existing biases. Even though some technology and regulation can help here fundamentally these require internal changes which I later describe in a section called psychological freedom. 
+For instance, there are a lot of people at the moment who feel offended by content that is available on the Internet. Many comment threads and forums have people yelling at each other. Others spend their time trapped within polarized online communities which confirm only their existing biases, a phenomenon so common that has been coined an &ldquo;echo chamber&rdquo; {{"ARSTECH" | cite}}. Even though some technology and regulation can help here, fundamentally these require internal changes which I later describe in a section called psychological freedom. 
```

Comment:
If we add something here it should be that online systems are designed to keep people in their "filter bubble" -- but "echo chamber" is a pre-Internet term